gemfiles/Gemfile.rails42.lock

```diff
@@ -0,0 +1,160 @@
+PATH
```

Comment:
Why do we need to have this lock file?

Reply:
Simply to force Github Actions to select Bundler 1.17.3
this is required for Rails 4.2 tests to run on Github Actions. or at least, its the best solution I found that I use for many gems.
we can delete once we decide to drop Rails 4.2 from the test suite.